streams/examples/src/test/java/org/apache/kafka/streams/examples/wordcount/WordCountProcessorTest.java

```diff
@@ -61,7 +61,7 @@ public void test() {
         context.scheduledPunctuators().get(0).getPunctuator().punctuate(0L);
 
         // finally, we can verify the output.
-        final Iterator<MockProcessorContext.CapturedForward> capturedForwards = context.forwarded().iterator();
+        final Iterator<MockProcessorContext.CapturedForward<String, String>> capturedForwards = context.forwarded().iterator();
```

Comment:
Since the new ProcessorContext bounds the forward calls, I've also added types to the captured forwards.